PyQt5-stubs/QtCore.pyi

```diff
@@ -9264,7 +9264,7 @@ def oct_(s: QTextStream) -> QTextStream: ...
 def bin_(s: QTextStream) -> QTextStream: ...
 def Q_RETURN_ARG(type: typing.Any) -> QGenericReturnArgument: ...
 def Q_ARG(type: typing.Any, data: typing.Any) -> QGenericArgument: ...
-def pyqtSlot(*types: typing.Any, name: typing.Optional[str] = ..., result: typing.Optional[str] = ...) -> typing.Callable[[FuncT], FuncT]: ...
+def pyqtSlot(*types: typing.Any, name: typing.Optional[str] = ..., result: typing.Type[object] = ...) -> typing.Callable[[FuncT], FuncT]: ...
```

Comment:
https://www.riverbankcomputing.com/static/Docs/PyQt5/signals_slots.html

Just noticed we're missing the `revision` argument